pages/app/components/theme-switcher.tsx

```diff
@@ -28,11 +43,21 @@ export default function ThemeSwitcher() {
   return (
     <SpaceBetween direction="horizontal" size="xs">
       <label>
-        Theme
+        Theme{' '}
         <select defaultValue={THEME}>
           <option value={THEME}>{THEME}</option>
         </select>
       </label>
+      <label>
```

Comment:
Adding an extra select here will fail too many screenshot tests. 

1. If it gets into merge, try to isolate this commit from others to make it easier to review in the pipeline
2. You can do team a huge favor and refactor the header to use modal for preferences (because their count exceeded reasonable limit, time to go with the modal)

Reply:
All preferences are exposed via url params. If any test is manually interacting with the toggle, this test can be simplified